.github/workflows/cypress.yml

```diff
@@ -0,0 +1,98 @@
+name: Cypress CI Workflow
+
+on:
+  push:
+    branches:
+      - feat/user-role-permissions
+      - ft/add-device-test
+  pull_request:
```

Comment:
can you remove the push part and for the pull_request part replace with the following branches
- develop
- stage
- prod